site/layouts/shortcodes/ods-recommendation-alert.html

```diff
@@ -0,0 +1,13 @@
+{{- /*
+Usage: `ods-recommendation-alert`
+*/ -}}
+
+<div class="bd-ods-alert alert alert-info">
```

Comment:
Do we need `role="alert"` (same as the danger one ?)

Reply:
No: in my opinion, we do not need alert role, neither on danger or info callout, because it iwill be vocalize first and out of context, which is not what we want for these callouts..